assets/wire/reducers.js

```diff
@@ -173,17 +175,34 @@ export default function wireReducer(state = initialState, action) {
             activeItem: action.item || null,
         };
 
-    case PREVIEW_ITEM:
+    case PREVIEW_ITEM: {
+        const readItems = Object.assign({}, state.readItems);
```

Comment:
this `readItems` handling is duplicated here and in OPEN_ITEM case, could use a single function to avoid duplication